thirdeye/thirdeye-frontend/app/pods/components/composite-anomalies/entity-metrics-anomalies/template.hbs

```diff
@@ -0,0 +1 @@
+{{composite-anomalies/data-table title=title tableData=tableData tableColumns=tableColumns noRecords=noRecords}}
```

Comment:
Sometimes the properties are on their own line in the template, but sometimes they are all on one line like this.  Is this because it's the only element in the handlebar file?

Reply:
Yeah, it is likely. But it is also possible that there are not enough attributes for the linter to deem it necessary to show the attributes on a new line. I am relying on prettier on handlebars for all the formatting, to take out this guesswork.